etl/steps/data/garden/epa/2024-01-29/ocean_heat_content.yml

```diff
@@ -0,0 +1,38 @@
+definitions:
+  common:
+    presentation:
+      topic_tags:
+      - Climate Change
+    processing_level: minor
+
+dataset:
+  title: Ocean Heat Content
+  update_period_days: 0
+
+tables:
+  ocean_heat_content_monthly:
+    title: Ocean Heat Content - Monthly average
+    variables:
+      ocean_heat_content_700m:
+        title: Monthly average ocean heat content for the 0-700 meters layer
```

Comment:
same here maybe more context from the source directly - e.g., common `description_from_producer: The amount of heat in the ocean, or ocean heat content, is an important indicator of climate change because the oceans ultimately absorb a large portion of the extra energy that greenhouse gases trap near the Earth’s surface. Ocean heat content also plays an important role in the Earth’s climate system because heat from ocean surface waters provides energy for storms and thereby influences weather patterns.`